### PR TITLE
Convert JSONRPCMessage command params to string

### PR DESCRIPTION
**Pull request by balloob**
https://github.com/home-assistant/core/issues/71101

### CODE REVIEW

File: aioslimproto/json_rpc.py [hunk 1]
```diff
@@ -33,24 +33,19 @@ class JSONRPCMessage:
...
+    command_args: list[str]
+
+    @classmethod
+    def from_json(  # pylint: disable=redefined-builtin
+        cls, id: int, method: str, params: list
+    ) -> JSONRPCMessage:
+        """Create a JSONRPCMessage from JSON."""
+        player_id = str(params[0])
+        command = str(params[1][0])
+        command_args = [str(v) for v in params[1][1:]]
```
[the remark this answers is not in the document]
Comment (balloob) [reply aioslimproto/json_rpc.py:47]: That works. Added test to verify.

File: aioslimproto/json_rpc.py [hunk 2]
```diff
@@ -142,7 +137,7 @@ async def _handle_client(
                 await self.send_response(writer, 405, "Method or path not allowed")
                 return
 
-            rpc_msg = JSONRPCMessage(**json.loads(body))
+            rpc_msg = JSONRPCMessage.from_json(**json.loads(body))
```
[the remark this answers is not in the document]
Comment (balloob) [reply aioslimproto/json_rpc.py:140]: Can we focus in this PR on the bug fix. We can refactor later.

[the remark this answers is not in the document]
Comment (marcelveldt) [reply aioslimproto/json_rpc.py:140]: Yeah it was just on my mind while reviewing this PR

**Review by marcelveldt — APPROVED** [review]
LGTM!

**Pull request merged**